style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: red;
+  text-align: center;
+}
```

Comment:
- [ ] Please make sure that you are using the class with styles for your `h1` instead of targeting it directly over the tag name.

Reply:
Thank you for your suggestion.